sycl/test-e2e/Matrix/joint_matrix_bfloat16_packedB.cpp

```diff
@@ -11,6 +11,8 @@
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
 // RUN: %if gpu %{ env IGC_JointMatrixLoadStoreOpt=2 %{run} %t.out %}
```

Comment:
If contiguous optimization does not get triggered for this test, we can remove `env IGC_JointMatrixLoadStoreOpt=2` option to save on run time; no need to run the same assembly twice.

Reply:
Actually there are cases, where contiguous optimization is triggered always, if the flag is 2 - it is for matrix C 1x64.
If we use flag = 1, then contiguous optimization will not be triggered for this combination and hence not tested.